### DISCUSSION TITLE
MacOS requirement

### DISCUSSION THREAD

**Discussion by dirtycold** [Uncommon/Xit#348] [Q&A] [answered]
Hi,

After upgrading Xit to 1.0b9 (via brew cask), Xit is unable to run on my Mac. It requires MacOS 10.13.
Will this requirement going to be a permanent change?
![xit](https://user-images.githubusercontent.com/1041980/42357115-d17fa720-8108-11e8-8c5c-51f9567d3085.png)

I am using MacOS Sierra 10.12.6.

**Comment by Uncommon** [comment]
It looks like 10.12 is doable; I'll just have to make some adjustments to the touch bar code which requires 10.12.2.

**Comment by Uncommon** [comment, marked as answer]
Actually, it turns out 10.12 is also missing `NSColor(named:)` which I need now that I've moved all the colors into an asset catalog, and there isn't a good backwards compatible solution for that. Sorry.

**Comment by Uncommon** [comment]
Side note: I didn't even know about `brew cask`, much less that there was an entry for Xit. Cool!

**Comment by dirtycold** [comment]
Ok, well understood. Looks like upgrading to 10.13 is a better idea. Thanks for creating Xit !

**Comment by Uncommon** [comment]
You're welcome! Please keep the feedback coming :)
